### ISSUE TITLE
search parameter viewer.html doesn't open automatically findbar with search string in text field

### ISSUE THREAD

**Issue by van-nes** [mozilla/pdf.js#9831]
A pdf.js viewer.html call with (e.g.) search parameter:
    www.dezijpe.nl/pdfjs/web/viewer.html?file=/zhb/pdf/2007030311.pdf#search=abc&phrase=true
works fine, BUT doesn't open automatically the findbar.

Only after using manually the magnifier icon (findbar.title), the findbar will be opened, however
with empty search field (missing the search string from: #search=abc).
Only the number of hits is correctly shown.

How exactly (implementation in viewer.js ?) to:
1. automatically open findbar (PDFViewerApplication.findBar.open(); ?)
2. automatically include the search text-string from param #serach =

Thanks for help.

**Comment by erzaehlsalex** [comment]
@van-nes Did you create a PR for that? I think this is a really good feature!
@timvandermeij This is a really small feature that could be merged easily, right? :)

**Comment by Snuffleupagus** [comment]
This is essentially a duplicate of issue #8185, and should be closed as such.

> This is a really small feature that could be merged easily, right? :)

Unfortunately not, since that completely ignores the Firefox integration; please refer to https://github.com/mozilla/pdf.js/issues/8185#issuecomment-288717592 and the discussion in #3565.

**Comment by Snuffleupagus** [comment]
@timvandermeij Mind closing this one, since it's a duplicate of issue #8185?
